src/backend/templates/schedule.hbs

```diff
@@ -539,9 +542,18 @@
                     $(sessionName).html(textArray.join(' ') + '...');
                 }
               })
+              roomCount++;
             });
+
             if(width < $(rooms).width()) {
-              $(rooms).width(width);
+              offset = ($(rooms).width() - width)/roomCount;
```

Comment:
@aayusharora We can go with what you are suggesting, But I'm not quite sure that would have any influence over the load speeds of the page. The function `adjustCalendar()` is only triggered when the user is in the calendar view and not `onLoad` of the page. If we really want to improve the load time of the page we should 

- Not execute the `display()` function onload. That would significantly help as on load we don't need to trigger it as it takes all the sessions of the page and checks if it should be displayed or not.

Please share your views on this.